openarena-q3e

```diff
@@ -1,29 +1,59 @@
 #!/bin/sh
-OA_CVARS="+set fs_basegame baseoa +set protocol 71"
-DISABLED_CVARS="+set com_protocol 72 +set cl_voip 1"
 
+###
+CONFIG_FILE=".var/app/io.github.ec_.Quake3e.OpenArena/config/cfg"
+# Check if the config file exists
+if [ -f "$CONFIG_FILE" ]; then
+    source "$CONFIG_FILE"
+fi
+# Check if it's the first run or if RESPECT_XDGDIRS is not set
+if [ ! -f "$CONFIG_FILE" ] || [ -z "$RESPECT_XDGDIRS" ]; then
+    RESPECT_XDGDIRS=0
+
+    # Save RESPECT_XDGDIRS in the config file
+    echo "RESPECT_XDGDIRS=$RESPECT_XDGDIRS" > "$CONFIG_FILE"
+fi
+# Use the value of RESPECT_XDGDIRS to determine the directory
+if [ "$RESPECT_XDGDIRS" -eq 1 ]; then
+    # Use ~/.var/app/io.github.ec_.Quake3e.OpenArena/.openarena/ directory
+    mkdir -p "$HOME/.var/app/io.github.ec_.Quake3e.OpenArena/homepath"
+    OA_DIR="$HOME/.var/app/io.github.ec_.Quake3e.OpenArena/homepath"
+    rm -rf "$HOME/.openarena"
+else
+    # Use .openarena directory
+    OA_DIR="$HOME/.openarena"
+    mkdir -p "$HOME/.var/app/io.github.ec_.Quake3e.OpenArena/homepath"
```

Comment:
Why creating the XDG directory if we use a default one?

Reply:
Well, I'm asking for your help, I'm not really good at this. When I get home I will make changes. I didn't know all that so thanks

Reply:
I will try to provide help, but I am quite busy with work, life and other things so please bear with me.